content/projects/gsoc/proposing-project-ideas.adoc

```diff
@@ -34,13 +34,53 @@ extending it by adding new features like link:/doc/book/pipeline/[Jenkins Pipeli
 * Creating new demo and reference setups,
 powered by various "-as-Code" engines (e.g. Jenkins Pipeline, JobDSL, Docker, link:/projects/gsoc/gsoc2018-project-ideas/#jenkins-configuration-as-code[Configuration-as-Code plugin], etc.)
 
-Once the project idea is proposed in the mailing list,
-link:/projects/gsoc/#mentors-and-org-admins[GSoC org admins] will help to finalize the idea and to reach out to potential mentors/co-mentors.
-
 == Notes for students
 
 Although we encourage students to propose their own project ideas, we cannot guarantee
 that will find potential mentors for every proposal, especially for narrow areas.
 During the selection phase we won't be able to accept proposals without mentors, so
 we highly recommend getting initial feedback in the mailing lists before spending too much
 time on such proposals.
+
+== Acceptance process for new project proposals
+
+Project proposals are tracked in two places. When first proposed they are added to
+the link:https://docs.google.com/document/d/14N6kCShmxy4SumT0khuEFxXYZE4v1_bimK66PJuBHzM/edit#heading=h.o5kqo7p5rgto[project
+  table in the projects doc]. Once a project has been deemed viable, it is also published to
+  link:https://jenkins.io/projects/gsoc/2019/project-ideas[jenkins.io].
+
+Projects are proposed by the project champion. The champion will be responsible for shepherding the
+proposal through the process. The acceptance process includes the following steps:
+
+. The project champion creates a project proposal and adds the proposal to the
```

Comment:
I think we should let champions to make the proposals in the GSoC mailing list or in Gitter to simplify the process for them. Table entry can be added by org admins